### ISSUE TITLE
[FE] Review feature flipper for accessibility feature flags that can be removed

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#53357]
## Background 

We've launched most of the accessibility updates we started as part of our 2022 profile accessibility audit. For any work already launched to prod, we should go through the feature flipper and remove an flags that are no longer needed.

## Tasks

- [x] Review feature flipper
- [x] Remove `profile_schema_forms`
- [x] Remove `profile_show_address_change_modal`
- [x] Remove `profile_use_security_process_list`
- [x] Remove `profile_block_for_fiduciary_deceased_or_incompetent`

## Acceptance criteria

- [x] Any accessibility-related/old feature flags have been removed
- [x] Any feature work around flags and related logic has been removed

**Comment by BerniXiongA6** [comment]
HI @adamwhitlock1 -- will this carry over to Sprint 5? And if so how many points left?

**Comment by adamwhitlock1** [comment]
I requested a review of the open PR from platform, so it should be able to be merged and validated today 🤞🏼

**Comment by adamwhitlock1** [comment]
I've reviewed this in staging, and all functionality is present, and the feature toggles are removed from all FE code.

Will close this now and revisit in case there is anything else that comes up.
